README.md

```diff
@@ -102,11 +108,13 @@ class AccountApproved extends Notification
 ### Available Message methods
 
 A list of all available options
-- `body('')`: Accepts a string value for the body.
-- `enableSound()`: Enables the notification sound.
-- `disableSound()`: Mutes the notification sound.
-- `badge(1)`: Accepts an integer value for the badge.
-- `ttl(60)`: Accepts an integer value for the time to live.
+
+-   `title('')`: Accepts a string value for the title.
+-   `body('')`: Accepts a string value for the body.
+-   `enableSound()`: Enables the notification sound.
+-   `disableSound()`: Mutes the notification sound.
+-   `badge(1)`: Accepts an integer value for the badge.
+-   `ttl(60)`: Accepts an integer value for the time to live. -`jsonData('')`: Accepts a json string or an array for additional. -`channelID('')`: Accepts a string to set the channelId of the notification for Android devices.
```

Comment:
`jsonData` should be in a separate line.

Reply:
I've committed those changes :) Thanks